pkg/runner/runner.go

```diff
@@ -728,6 +721,12 @@ func GetEntryCodeInstructions(function starknet.EntryPointByFunction, proofmode
 	for _, retArgs := range function.ReturnArgs {
 		adjustedRetOffset += retArgs.Size
 	}
+
+	// builtins have to be ordered by the highest id to generate proper offsets
+	for i, j := 0, len(function.Builtins)-1; i < j; i, j = i+1, j-1 {
+		function.Builtins[i], function.Builtins[j] = function.Builtins[j], function.Builtins[i]
+	}
```

Comment:
Use [`slice.Reverse`](https://cs.opensource.google/go/go/+/refs/tags/go1.21.0:src/slices/slices.go#:~:text=493-,494,-495) instead. It does the exact same thing

Reply:
thanks! way cleaner